### PR TITLE
Add org.gajim.Gajim_nightly

### PR DESCRIPTION
**Pull request by a-andre**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

**Comment by a-andre** [comment]
@lovetox FYI

**Comment by LeandroStanger** [comment]
bot, build org.gajim.Gajim_nightly

**Comment by hfiguiere** [comment]
Flathub is not for nightly build.

File: org.gajim.Gajim_nightly.yaml [hunk 1]
```diff
@@ -0,0 +1,331 @@
...
+  - name: gajim
+    buildsystem: simple
+    build-commands:
+      - pip3 install .
+      - touch /app/share/run-as-flatpak
+      - echo -e '#!/bin/sh\ngajim --gapplication-app-id=org.gajim.Gajim_nightly "$@"' > /app/bin/gajim-devel
+      - chmod 755 /app/bin/gajim-devel
+    sources:
+      - type: git
+        url: https://dev.gajim.org/gajim/gajim.git
```
Comment (hfiguiere) [inline org.gajim.Gajim_nightly.yaml:323]: See:
https://github.com/flathub/flathub/issues/2975#issuecomment-1086897335

Flathub is not for nightly builds.

**Pull request closed without merging**